Description: Apply LDFLAGS to linker
Author: Jonas Smedegaard <dr@jones.dk>
Forwarded: no
License: GPL-2+
Last-Update: 2015-05-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Makefile
+++ b/Makefile
@@ -36,7 +36,7 @@
 	$(CC) $(CFLAGS) $(DEBUG) $(OPTIMIZE) -c -o app_flite.o app_flite.c
 
 app_flite.so: app_flite.o
-	$(CC) -shared -Xlinker -x -o $@ $< $(LIBS)
+	$(CC) -shared $(LDFLAGS) -Xlinker -x -o $@ $< $(LIBS)
 
 clean:
 	rm -f app_flite.o app_flite.so
